package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -303,20 +307,24 @@ class DiffusionMap(object):
 
     Methods
     -------
-    run()
+    run()line
         Constructs an anisotropic diffusion kernel and performs eigenvalue
         decomposition on it.
     transform(n_eigenvectors, time)
         Perform an embedding of a frame into the eigenvectors representing
         the collective coordinates.
+    
+    .. versionchanged: 2.2.0
```

Comment:
See here: https://mdanalysis--3541.org.readthedocs.build/en/3541/documentation_pages/analysis/diffusionmap.html#MDAnalysis.analysis.diffusionmap.DiffusionMap.transform

You'll need two empty lines for this to work as intended.
```suggestion

    .. versionchanged: 2.2.0
```

Reply:
hi, I'm not very sure why it still doesn't work as intended even after giving 2 empty lines.